dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/IndexStats.java

```diff
@@ -0,0 +1,34 @@
+package com.dotcms.content.elasticsearch.business;
+
+import org.elasticsearch.common.unit.ByteSizeValue;
+
+public class IndexStats {
+
+    private String indexName;
+    private int documentCount;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'documentCount' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)